scripts/start-server.sh

```diff
@@ -53,17 +53,13 @@ else
         +quit
     fi
 fi
-BEPINEX_VR_TS_API_URL=https://thunderstore.io/c/v-rising/api/v1/package/b86fcaaf-297a-45c8-82a0-fcbd7806fdc4/
-BEPINEX_VR_TS_URL=https://v-rising.thunderstore.io/package/BepInEx/BepInExPack_V_Rising/
-BEPINEX_VR_TS_JSON=${SERVER_DIR}/BepInExPack_V_Rising.json
 
 if [ "${ENABLE_BEPINEX}" == "true" ]; then
-    export  WINEDLLOVERRIDES="winhttp=n,b"
```

Comment:
Without this BepInEx won't be loaded. It's needed for wine to pick up the winhttp.dll in the V Rising folder and load BepInEx.

Reply:
Thank you for the hint, adding the line again.
Will update the container shortly.